Lib/test/test_struct.py

```diff
@@ -618,6 +618,14 @@ def test_issue29802(self):
         # Shouldn't crash.
         self.assertEqual(struct.unpack(b'b', b'a'), (b'a'[0],))
 
+    def test_format_attr(self):
+        s = struct.Struct('=i2H')
+        self.assertEqual(s.format, '=i2H')
```

Comment:
IMO, `self.assertIsInstance(s.format, str)` would be better to understand (for future readers) the behavior change.